.travis.yml

```diff
@@ -27,25 +27,23 @@ matrix:
     # NOTE: We combine builds because Travis offers a maximum of 5 concurrent
     # builds and having 6 tasks / builds means 1 tasks will need to wait for one
     # of the other 5 tasks to finish before it can start
-    #- env: TASK=ci-unit NODE_INDEX=0 NODE_TOTAL=2
-    #  python: 2.7
-    #  name: "Unit Tests (Python 2.7) - 1"
-    #- env: TASK=ci-unit NODE_INDEX=1 NODE_TOTAL=2
-    #  python: 2.7
-    #  name: "Unit Tests (Python 2.7) - 2"
-    - env: TASK=ci-unit
+    # NOTE: Thresholds only apply to the make target and not to the whole build
+    # job which also includes "make requirements" and other steps
+    # "make requirements" can take substantially lower if the cache is purged
+    # and this would cause too many intermediate failures / false positives
+    - env: TASK=ci-unit CACHE_NAME=py2 COMMAND_THRESHOLD=670
       python: 2.7
       name: "Unit Tests (Python 2.7)"
-    - env: TASK=ci-integration
+    - env: TASK=ci-integration CACHE_NAME=py2 COMMAND_THRESHOLD=670
```

Comment:
How did we arrive with the value of 670, 270, 310, etc?

Reply:
I took at look at the current build time averages for each task and added 30-60 seconds (so we have some grace because build times can very quite a bit depending on how the infrastructure is utilized).